conf/fluo-deploy.props.example

```diff
@@ -54,6 +54,8 @@ default.instance.type = m3.large
 key.name = my_aws_key
 # Tags to add instances
 #instance.tags = key1:value1,key2:value2
+#Number of ephemeral drives to setup.  Do not set higher than supported by instance type.
+#ephemeral.num = 0
```

Comment:
>  sets the number of ephemeral drives for the whole cluster which will cause problems if some nodes have 2 drives and others have 1 drive

What about documenting that this property can not exceed the minimum of # ephemeral supported on all instance types used?

> However, letting nodes have different numbers of drives for different nodes can make things complicated

Agreed

>  It might be nice to remove the ephemeral.num property and just keep a map of instance types to number of ephemeral drives.

What about the case where a node supports up to 8 ephemeral drives and I only want to use 4?  Not sure this is a realistic use case, but something to consider.